app/assets/stylesheets/base/_forms.scss

```diff
@@ -72,7 +72,12 @@ input[type="file"] {
 }
 
 select {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.